docs/tech-notes/jobs.md

```diff
@@ -0,0 +1,474 @@
+# The Job System
+
+Original authors: Michael Butler & Shiranka Miskin (November 2021)
+
+
+
+## Introduction
+
+Jobs are CockroachDB's way of representing long running background tasks.  They
+are used internally as a core part of various features of CockroachDB such as
+Changefeeds, Backups, and Schema Changes .  Progress is regularly persisted such
+that it can be resumed on node failure, the tasks can be
+paused/resumed/cancelled through SQL commands such as [`PAUSE
+JOB`](https://www.cockroachlabs.com/docs/stable/pause-job.html), and
+running/recent jobs can be inspected such as through the [`SHOW
+JOBS`](https://www.cockroachlabs.com/docs/stable/show-jobs.html) command.  They
+can even be scheduled to run periodically through commands such as [`CREATE
+SCHEDULE FOR BACKUP`
+](https://www.cockroachlabs.com/docs/stable/create-schedule-for-backup.html#synopsis).
+
+
+
+## What is a Job?
+
+A Job is represented as a row in the
+[`system.jobs`](https://github.com/cockroachdb/cockroach/blob/7097a9015f1a09c7dee4fbdbcc6bde82121f657b/pkg/sql/catalog/systemschema/system.go#L185)
+table.  This table is the single source of truth for the information about
+currently relevant jobs.  Nodes read this table and claim jobs that are
+unclaimed and unfinished, using the [`payload`](https://github.com/cockroachdb/cockroach/blob/8a501a247f177bf287bcf34beb4f05155818998c/pkg/jobs/jobspb/jobs.proto#L755)
```

Comment:
The word `claim` is a strange one to see so early on.  I think you need to introduce job execution section, where you explain what claimed jobs are, some inner details of registry, etc.